src/main/java/com/puppycrawl/tools/checkstyle/api/JavadocTokenTypes.java

```diff
@@ -313,7 +313,7 @@
      *       |--SEE_LITERAL[3x0] : [@see]
      *       |--WS[3x4] : [ ]
      *       |--REFERENCE[3x5] : [org.apache.utils.Lists.Comparator#compare(Object)]
-     *           |--PACKAGE[3x5] : [org.apache.utils]
+     *           |--PACKAGE_CLASS[3x5] : [org.apache.utils]
      *           |--DOT[3x21] : [.]
      *           |--CLASS[3x22] : [Lists]
      *           |--DOT[3x27] : [.]
```

Comment:
These javadoc tress and others alike present in other java files need to be updated to be synchronized with current checkstyle version once the currently prioritized ANTLR issues are resolved.

It doesn't seem efficient to update them in this PR and then keep updating them with every other ANTLR issue in line. Will Create an issue for it once done with ANTLR changes